inc/Parsing/Handler/Quote.php

```diff
@@ -101,6 +73,10 @@ public function process()
         return $this->callWriter;
     }
 
+    /**
+     * @param $marker
```

Comment:
Since `preg_match` takes only a `string` as subject, shouldn't `$marker` be defined as string at least in the phpdoc?
```suggestion
     * @param string $marker
```
(Sorry for the unsolicited comment, but I'm currently looking into this after the upgrade to the latest beta, and I'm happy you're already addressing the access issue!)